exchange/bidder_test.go

```diff
@@ -1568,6 +1568,46 @@ func TestMobileNativeTypes(t *testing.T) {
 	}
 }
 
+func TestAddNativeTypes(t *testing.T) {
```

Comment:
I wonder if there should be more cases with this test just to ensure that the slight pointer changes made in this PR don't mess anything else up within `addNativeTypes`. What do you think?